atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownGetTest.java

```diff
@@ -0,0 +1,145 @@
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.cassandra.multinode;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+import java.util.SortedMap;
+
+import org.junit.Test;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Multimap;
+import com.palantir.atlasdb.encoding.PtBytes;
+import com.palantir.atlasdb.keyvalue.api.BatchColumnRangeSelection;
+import com.palantir.atlasdb.keyvalue.api.Cell;
+import com.palantir.atlasdb.keyvalue.api.ColumnSelection;
+import com.palantir.atlasdb.keyvalue.api.RangeRequest;
+import com.palantir.atlasdb.keyvalue.api.RowColumnRangeIterator;
+import com.palantir.atlasdb.keyvalue.api.RowResult;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.keyvalue.api.Value;
+import com.palantir.common.base.ClosableIterator;
+
+
+public class OneNodeDownGetTest {
+
+    @Test
+    public void canGet() {
+        verifyTimestampAndValue(OneNodeDownTestSuite.CELL_1_1, OneNodeDownTestSuite.DEFAULT_TIMESTAMP,
+                OneNodeDownTestSuite.DEFAULT_VALUE);
+    }
+
+    @Test
+    public void canGetRows() {
+        Map<Cell, Value> row = OneNodeDownTestSuite.db.getRows(OneNodeDownTestSuite.TEST_TABLE,
+                ImmutableList.of(OneNodeDownTestSuite.FIRST_ROW),
+                ColumnSelection.all(), Long.MAX_VALUE);
+
+        assertEquals(OneNodeDownTestSuite.DEFAULT_TIMESTAMP, row.get(OneNodeDownTestSuite.CELL_1_1).getTimestamp());
+        assertEquals(new String(OneNodeDownTestSuite.DEFAULT_VALUE),
+                new String(row.get(OneNodeDownTestSuite.CELL_1_1).getContents()));
+        assertEquals(OneNodeDownTestSuite.DEFAULT_TIMESTAMP, row.get(OneNodeDownTestSuite.CELL_1_2).getTimestamp());
+        assertEquals(new String(OneNodeDownTestSuite.DEFAULT_VALUE),
+                new String(row.get(OneNodeDownTestSuite.CELL_1_2).getContents()));
+    }
+
+    @Test
+    public void canGetRange() {
+        final RangeRequest range = RangeRequest.builder().endRowExclusive(OneNodeDownTestSuite.SECOND_ROW).build();
+        ClosableIterator<RowResult<Value>> it = OneNodeDownTestSuite.db.getRange(OneNodeDownTestSuite.TEST_TABLE, range,
+                Long.MAX_VALUE);
+
+        RowResult<Value> row = it.next();
```

Comment:
I'd construct an expected RowResult:

```
RowResult<Value> expectedRow = RowResult.of(...);
assertThat(it).containsOnly(expectedRow);
```

Here and throughout: you can use the iterable assertions of assertj (see [examples](https://github.com/joel-costigliola/assertj-examples/blob/master/assertions-examples/src/test/java/org/assertj/examples/IterableAssertionsExamples.java)), and rely on the equality methods of `Value` rather than extracting the values, constructing strings and comparing them, etc.